lib/api.js

```diff
@@ -2133,4 +2133,25 @@ API.prototype.pushNotificationsUnsubscribe = function(cb) {
   });
 };
 
+/**
+ * Returns result object.
+ * @param {String} opts
+ * @param {Int} opts.feePerKb - Fee value
+ * @param {String} opts.excludeUnconfirmedUtxos - Indicates it if should use (or not) the unconfirmed utxos
+ * @param {String} opts.returnInputs - Indicates it if should return (or not) the inputs
+ * @return {Callback} cb - Return error (if exists) and object result
+ */
+API.prototype.getSendMaxInfo = function(opts, cb) {
+  var self = this;
+  var url = '/v1/sendmaxinfo/';
+  if (opts.feePerKb) url += '?feePerKb=' + opts.feePerKb;
+  if (opts.excludeUnconfirmedUtxos) url += '&excludeUnconfirmedUtxos=' + opts.excludeUnconfirmedUtxos;
+  if (opts.returnInputs) url += '&returnInputs=' + opts.returnInputs;
```

Comment:
Both `excludeUnconfirmedUtxos` and `returnInputs` arguments should be boolean. This needs to be translated to '1' or '0' to be passed to the REST method.